configs/permissions/integrations.json

```diff
@@ -0,0 +1,4 @@
+{
+    "*": ["*"],
+    "endpoints": ["read","write"]
```

Comment:
Just to clarify expectations, this new file would be creating the following permissions:
```
integrations:*:*
integrations:endpoints:read
integrations:endpoints:write
```

Reply:
Correct (we're not using those yet, but should at some point)